src/main/scala/org/zalando/kanadi/api/Subscriptions.scala

```diff
@@ -1452,7 +1452,7 @@ case class Subscriptions(baseUri: URI, authTokenProvider: Option[AuthTokenProvid
       modifySourceFunction: Option[
         Source[SubscriptionEvent[T], UniqueKillSwitch] => Source[SubscriptionEvent[T], UniqueKillSwitch]])(implicit
       decoder: Decoder[List[Event[T]]],
-      flowId: FlowId = randomFlowId(),
+      flowId: FlowId,
```

Comment:
This is due to 

```
[warn] /Users/mdedetrich/github/kanadi/src/main/scala/org/zalando/kanadi/api/Subscriptions.scala:1455:7: private default argument in class Subscriptions is never used
[warn]       flowId: FlowId = randomFlowId(),
```

The default parameter is never used (since we always pass in `flowId` directly) and the method is not public.